challenges.json

```diff
@@ -142,6 +142,9 @@
     "level": 1,
     "name": "dead-mans-switch",
     "label": "Dead Mans Switch",
+    "repo": "https://github.com/KcPele/eth-tech-tree-challenges",
```

Comment:
Change this to "https://github.com/BuidlGuidl/eth-tech-tree-challenges".

My example above steered you wrong. I need to update the token-wrapper-weth challenge.